src/Searchable.php

```diff
@@ -15,6 +15,7 @@
 use SilverStripe\ORM\DataList;
 use SilverStripe\ORM\DataObject;
 use SilverStripe\ORM\DataObjectSchema;
+use SilverStripe\ORM\ValidationException;
 use SilverStripe\Versioned\Versioned;
 use Symbiote\QueuedJobs\Services\QueuedJobService;
 use function is_numeric;
```

Comment:
Really? If we start off down this rabbit hole then we'd be adding `use is_array`, `use isset` etc. Just lose it :-)

Reply:
Fixed